apps/web/src/pages/studies/[studyId]/index.tsx

```diff
@@ -100,50 +100,62 @@ export default function Study({ user, study, assessments }: StudyProps) {
                 <Table.Cell>
-                  {study.evaluationCategories.length
-                    ? study.evaluationCategories.map((evalCategory) => evalCategory.indicatorValue).join(', ')
+                  {studyInCPMS.StudyEvaluationCategories.length
+                    ? studyInCPMS.StudyEvaluationCategories.map(
+                        (evalCategory) => evalCategory.EvaluationCategoryValue
+                      ).join(', ')
                     : 'This study is progressing as planned'}
                 </Table.Cell>
               </Table.Row>
               <Table.Row>
                 <Table.CellHeader className="w-1/3">Planned opening date</Table.CellHeader>
-                <Table.Cell>{study.plannedOpeningDate ? formatDate(study.plannedOpeningDate) : '-'}</Table.Cell>
+                <Table.Cell>
+                  {studyInCPMS.PlannedClosureToRecruitmentDate ? formatDate(studyInCPMS.PlannedOpeningDate) : '-'}
+                </Table.Cell>
               </Table.Row>
               <Table.Row>
                 <Table.CellHeader className="w-1/3">Actual opening date</Table.CellHeader>
-                <Table.Cell>{study.actualOpeningDate ? formatDate(study.actualOpeningDate) : '-'}</Table.Cell>
+                <Table.Cell>
+                  {studyInCPMS.ActualOpeningDate ? formatDate(studyInCPMS.ActualOpeningDate) : '-'}
+                </Table.Cell>
               </Table.Row>
               <Table.Row>
                 <Table.CellHeader className="w-1/3">Planned closure to recruitment date</Table.CellHeader>
-                <Table.Cell>{study.plannedClosureDate ? formatDate(study.plannedClosureDate) : '-'}</Table.Cell>
+                <Table.Cell>
+                  {studyInCPMS.PlannedClosureToRecruitmentDate
+                    ? formatDate(studyInCPMS.PlannedClosureToRecruitmentDate)
+                    : '-'}
+                </Table.Cell>
               </Table.Row>
               <Table.Row>
                 <Table.CellHeader className="w-1/3">Actual closure to recruitment date</Table.CellHeader>
-                <Table.Cell>{study.actualClosureDate ? formatDate(study.actualClosureDate) : '-'}</Table.Cell>
+                <Table.Cell>
+                  {studyInCPMS.ActualOpeningDate ? formatDate(studyInCPMS.ActualClosureToRecruitmentDate) : '-'}
+                </Table.Cell>
               </Table.Row>
-              {study.studyStatus === 'Suspended' && Boolean(study.evaluationCategories.length) && (
+              {studyInCPMS.StudyStatus === 'Suspended' && Boolean(studyInCPMS.StudyEvaluationCategories.length) && (
                 <Table.Row>
                   <Table.CellHeader className="w-1/3">Estimated reopening date</Table.CellHeader>
                   <Table.Cell>
-                    {study.evaluationCategories[0].expectedReopenDate
-                      ? formatDate(study.evaluationCategories[0].expectedReopenDate)
+                    {studyInCPMS.StudyEvaluationCategories[0].ExpectedReopenDate
```

Comment:
I think this might throw an error if `StudyEvaluationCategories` is empty. 

Adding optional chaining should do the trick i.e. `studyInCPMS.StudyEvaluationCategories[0]?.ExpectedReopenDate`

Reply:
updated